lib/ansible/modules/packaging/os/pulp_repo.py

```diff
@@ -64,7 +65,8 @@
         the file content or the path to the file.
     type: str
     aliases: [ importer_ssl_client_cert ]
-  client_key:
+  feed_client_key:
+    version_added: "2.10"
     description:
```

Comment:
You should mention that the default value is taken from `client_key` if that's specified, and that this will change in Ansible 2.14.